modules/appeals_api/app/serializers/appeals_api/evidence_submission_serializer.rb

```diff
@@ -10,17 +10,17 @@ class EvidenceSubmissionSerializer < ActiveModel::Serializer
 
     attributes :id, :status, :code, :detail, :appeal_type, :appeal_id, :location, :created_at, :updated_at
 
+    delegate :status, to: :object
+    delegate :code, to: :object
+
     def id
       object.guid
     end
 
-    delegate :status, to: :object
-    delegate :code, to: :object
-
     def detail
       details = object.detail.to_s
       details = "#{details[0..MAX_DETAIL_DISPLAY_LENGTH - 1]}..." if details.length > MAX_DETAIL_DISPLAY_LENGTH
-      details
+      details.presence
```

Comment:
Personally I think that a guard statement would make sense here (maybe something like `return unless object.detail`). Otherwise you have to read the entire method and remember how `String#presence` works to know the outcome for what should be the majority of cases. Thoughts?